plugins/modules/memorydb_acl.py

```diff
@@ -0,0 +1,214 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+# Copyright: (c) 2022, Ansible Project
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+# template: header.j2
+# This module is autogenerated using the ansible.content_builder.
+# See: https://github.com/ansible-community/ansible.content_builder
+
+
+DOCUMENTATION = r"""
+module: memorydb_acl
+short_description: Creates and manages a MemoryDB ACL
+description:
+- Creates and manages an Access Control List (ACL) to specify permissions to a MemoryDB
+    for Redis cluster. For more information, see U(https://docs.aws.amazon.com/memorydb/latest/devguide/clusters.acls.html)
+options:
+    acl_name:
+        description:
+        - The name of the acl.
+        type: str
+    force:
+        default: false
+        description:
+        - Cancel IN_PROGRESS and PENDING resource requestes.
+        - Because you can only perform a single operation on a given resource at a
+            time, there might be cases where you need to cancel the current resource
+            operation to make the resource available so that another operation may
+            be performed on it.
+        type: bool
+    purge_tags:
+        default: true
+        description:
+        - Remove tags not listed in I(tags).
+        type: bool
+    state:
+        choices:
+        - present
+        - absent
+        - list
+        - describe
+        - get
+        default: present
+        description:
+        - Goal state for resource.
+        - I(state=present) creates the resource if it doesn't exist, or updates to
+            the provided state if the resource already exists.
+        - I(state=absent) ensures an existing instance is deleted.
+        - I(state=list) get all the existing resources.
+        - I(state=describe) or I(state=get) retrieves information on an existing resource.
+        type: str
+    tags:
+        aliases:
+        - resource_tags
+        description:
+        - A dict of tags to apply to the resource.
+        - To remove all tags set I(tags={}) and I(purge_tags=true).
+        type: dict
+    user_names:
+        $comment: List of users.
```

Comment:
@abikouo Time by time, the API is updated with new fields and the generator must be updated to ignore those fields if they are not relevant. You can manually remove those and open an issue in the generator if you wish.